dataland-inbound-proxy/templates/http-dataland-development.template

```diff
@@ -44,8 +44,17 @@ server {
       include utils/apiErrors.conf;
     }
 
-    location /internal-storage/actuator/health {
-      proxy_pass http://internal-storage:8080/internal-storage/actuator/health;
+    location /api/internal/ {
+        return 301 https://${PROXY_PRIMARY_URL}/nocontent;
+    }
+
+    location /internal-storage/ {
+      proxy_pass http://internal-storage:8080/internal-storage/;
```

Comment:
why do we need to expose the _internal_ storage to the outside?

Reply:
I think this was added to have no need to expose a port outside of the docker network and to have the internal storage accessible from the backend, running on the host machine locally.

Reply:
But this is also included in the prod-template, isn't it?

Reply:
it's not. I see. 
So this is fine. thanks for the explanation.